src/execution/execute.ts

```diff
@@ -163,6 +165,8 @@ export interface StreamUsage {
   fieldGroup: FieldGroup;
 }
 
+type GraphQLResult<T> = [T, ReadonlyArray<IncrementalDataRecord>];
```

Comment:
Maybe call this `GraphQLWrappedResult`, and use `wrappedResult` instead of `acc`?

Reply:
done! 24136afec